shell/list/node.vue

```diff
@@ -41,30 +40,30 @@ export default {
   },
 
   async fetch() {
-    const hash = { kubeNodes: this.$fetchType(NODE) };
+    this.$initializeFetchData(this.resource);
```

Comment:
The `this.resource` in this case is the `NODE` type.

It in fact get's called on line 66 with an `await` so that the mechanism for the loading status get's triggered `$fetchState.pending`.

The new way for fetching the resources works (with secondary resource fetches) needs the `resource-fetch` mixin to be initialized ( `this.$initializeFetchData(this.resource)` ) with the `primary` resource so that it populates the mixin variables with the correct data for the `watch`, `incremental` and other flags.

Since the mixin is included in this page it's only instanciated once and it's flags will be defined on the first run and subsequent runs should not affect them, from the changes I did to it.

After this initialization we are now free to dispatch all the `findAll's` that we need in parallel with blocking the interface.

Hence why we fire the secondary resources first and in the end we go for the `await` while fetching the primary resource.

This is the new pattern to consider when reviewing these changes.